x-pack/plugins/apm/public/hooks/use_breakpoints.ts

```diff
@@ -24,17 +28,19 @@ export function getScreenSizes(windowWidth: number) {
   };
 }
 
-export function useBreakPoints() {
+export function useBreakpoints() {
```

Comment:
we should move this hook to observability plugin or better to kibana_react package.

Reply:
Perhaps, but EUI has `useIsWithinBreakpoints` as well, and I find the return values from this hook confusing. I think we should rethink its API and see if the EUI team has suggestions before moving it.